src/Codeception/Util/Fixtures.php

```diff
@@ -32,9 +32,15 @@ public static function get($name)
         return self::$fixtures[$name];
     }
 
-    public static function cleanup()
+    public static function cleanup($name = null)
     {
-        self::$fixtures = [];
+        if ($name === null) {
+            self::$fixtures = [];
+        } elseif (!self::exists($name)) {
+            throw new \RuntimeException("$name not found in fixtures");
+        }
+
+        unset(self::$fixtures[$name]);
```

Comment:
so this unsets value even when `self::$fixtures = [];` ? 
please use `return`
Also I don't think the exception should be thrown if a key is not there. It means that it already empty, so cleaned up

Reply:
Sorry, late response. I'll update my merge request by the following:

```php
public static function cleanup($name = null) {
    if (self::exists($name)) {
        unset(self::$fixtures[$name]);
        return;
    }

     self::$fixtures = [];
}
```

Reply:
https://github.com/Codeception/Codeception/pull/5124#issuecomment-565353860